src/ray/raylet/node_manager.cc

```diff
@@ -667,16 +672,19 @@ void NodeManager::HandleRemoteDependencyCanceled(const ObjectID &dependency_id)
   // TODO(swang): Cancel reconstruction of the object.
 }
 
-void NodeManager::QueueTask(const Task &task) {
+// Check the task's dependencies and enqueue to either be ready for dispatch or waiting.
+// Called with tasks for which a placement decision has been made.
+void NodeManager::TransitionPlaceableTask(const Task &task) {
```

Comment:
Tasks passed to this method are in no queue. If tasks are left in the `PLACEABLE` queue, then perhaps `MoveTasks` could be further utilized to also transition tasks from `PLACEABLE` to `WAITING` or `READY`.